Alexander96779/bc-frontend
# Feature view specific incident Done!

#### What does this PR do?
- This PR allows users to view a single incident and check out all the details.
#### Description of Task to be completed?
- Create view specific incident component and connect it to redux.
- Design and style view specific incident page.
- Work on the redux part of the feature, use `Axios` to fetch data from the backend.
- Check the functionality of the feature.
#### How should this be manually tested?
- Clone this repo to your local machine, and go to `ft-view-specific-incident-UI` branch.
- Run `npm install` to install all the dependencies.
- Run `npm start` to start the app and use any browser to check this feature.
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- N/A
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/52447234/88697495-f349dd00-d104-11ea-8745-812ba80fe2a4.png)
